.github/workflows/ci.yml

```diff
@@ -55,7 +58,7 @@ jobs:
       with:
         persist-credentials: false
     - name: Set up Windows Builders
-      if: startswith(matrix.os, 'windows')
+      if: startswith(matrix.os, 'windows-x86_64') # FIXME: aarch64 doesn't have D:\ yet
```

Comment:
Maybe the AArch64 VMs are different enough that it just doesn’t matter, but one option would be [setting up a Dev Drive](https://github.com/samypr100/setup-dev-drive). It seemed slower than the native `D:\` when I was messing with it for x86, but maybe it’d be a win on AArch64.